### ISSUE TITLE
Issues on SL Beta

### ISSUE THREAD

**Issue by flow0284** [Caerdon/CaerdonWardrobe#3]
```
2x CaerdonWardrobe\external\ElvUI\Core.ElvUI.lua:114: CaerdonWardrobe\Core.lua:1199: attempt to index local 'color' (a nil value)
[string "=[C]"]: in function `error'
[string "@CaerdonWardrobe\external\ElvUI\Core.ElvUI.lua"]:114: in function <CaerdonWardrobe\external\ElvUI\Core.ElvUI.lua:109>

Locals:
(*temporary) = "CaerdonWardrobe\Core.lua:1199: attempt to index local 'color' (a nil value)"
```

Using ElvUI beta github builds

**Comment by yoshimo** [comment]
There is more before that:

Message: ..\AddOns\CaerdonWardrobe\Core.lua line 2071:
   attempt to call global 'GetQuestLogSelection' (a nil value)

**Comment by Caerdon** [comment]
Just released v2.5.0 which provides support for Shadowlands!
